resources/cert-manager/README.md

```diff
@@ -0,0 +1,61 @@
+# cert-manager
+
+## Overview
+
+`cert-manager` is a version of [Jetstack's cert-manager](https://cert-manager.io) customized to Kyma's needs.
+
+Quoting from the [official README](https://github.com/jetstack/cert-manager):
+
+>cert-manager is a Kubernetes addon to automate the management and issuance of
+>TLS certificates from various issuing sources.
+>
+>It will ensure certificates are valid and up to date periodically, and attempt
+>to renew certificates at an appropriate time before expiry.
+
+
+For more information check [links](#links) section.
+
+## Prerequisites
+
+- Kubernetes 1.11+
+
+## Details
+
+### Changes introduced
+
+The `crds.yaml` file has been split into multiple files. Each of these contains exactly one CRD with its name indicating the resource type. They are available in the `files/` directory.
+
+### Installation
+
+To install `cert-manager`, use the Kyma Installer and follow these steps:
+
+1. Manually apply CRDs from the `files/` directory:
+
+   ```bash
+   # Kubernetes 1.15+
+   $ kubectl apply -f ./files
+   ```
+
+2. Update the appropriate Installer CR template in `installation/resources/` by adding the following lines to the `spec.components` section:
+
+    ```yaml
+    - name: "cert-manager"
+      namespace: "cert-manager"
+    ```
+
+### Usage
+
+Before you configure `cert-manager` to issue certificates, you must first create Issuer resources. To learn more, read about [Issuers configuration](https://cert-manager.io/docs/configuration/).
+
+One of the common use cases for `cert-manager` is securing ingress resources. Read the [Securing Ingresses Resources documentation](https://cert-manager.io/docs/usage/ingress/) for more details.
+
+For more information, check the [official `cert-manager` documentation](https://cert-manager.io/docs/).
+
+## Troubleshooting
+
+One potential issue may occur while creating an Issuer or ClusterIssuer resources. It might result in failed calling webhook "webhook.cert-manager.io". A workaround that can be applied in this case is to delete both cert-manager's webhooks:
```

Comment:
```suggestion
One potential issue may occur while creating Issuer or ClusterIssuer resources. It might result in a failed call to the `webhook.cert-manager.io` webhook. A workaround for this is to delete these two `cert-manager`'s webhooks:
```
- The `an` would refer to the main noun, but the main noun (`resources`) is in plural, so it should be removed :) 
Unless you wrote it as `an Issuer's resources`, then `an` would refer to the Issuer.

- `that can be applied in this case` -> `for this`, nothing wrong with the way you wrote it, but this way it's shorter and conveys the same message :)

- `both` -> `these two` or `these`, unless the `cert-manager` has only two webhooks, then `both` is also fine.